### PR TITLE
Add support for tags per request: InVisionApp fork

### PR DESCRIPTION
**Pull request by mnichols**
This implements the changes requested in the PR [here](https://github.com/segmentio/stats/pull/100) but from our org repo. 
Thanks for the feedback!

### CODE REVIEW

File: go.mod [hunk 1]
```diff
@@ -1,4 +1,4 @@
-module github.com/segmentio/stats
+module github.com/InVisionApp/stats
```
Comment (maxcnunes) [inline go.mod]: @mnichols it looks like you need to revert this part

Comment (mnichols) [reply go.mod]: ah nice catch...fixed! @maxcnunes

**Comment by achille-roussel** [comment]
Thanks for the contribution!

**Comment by mnichols** [comment]
You're welcome! Here is a PR that has tests and docs for this ... you merged it before I could update the branch 😄 . https://github.com/segmentio/stats/pull/105

**Pull request merged**